src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,74 @@
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  selectedTodo: Todo,
+  handleCloseModal: () => void,
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo,
+  handleCloseModal,
+}) => {
+  const [todoOwner, setTodoOwner] = useState<User | null>(null);
+
+  const loadUser = useCallback(async () => {
+    const user = await getUser(selectedTodo.userId);
+
+    setTodoOwner(user);
+  }, []);
+
+  useEffect(() => {
+    loadUser();
+  }, []);
```

Comment:
In my opinion, you can write fetching logic inside useEffect, so you can get rid of useCallback:
```suggestion
  useEffect(() => {
    getUser(selectedTodo.userId)
      .then (user => setTodoOwner(user))
  }, []);
```